src/homework/day8/collections/MyBubbles.java

```diff
@@ -0,0 +1,36 @@
+package homework.day8.collections;
+
+import homework.day8.practice.Bubble;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class MyBubbles {
+
+    public static void main(String[] args) {
+
```

Comment:
Extra new line. Remove. The same for such cases.

Reply:
Extra line is removed.